Doc/library/typing.rst

```diff
@@ -1000,7 +1000,7 @@ These are not used in annotations. They are building blocks for creating generic
     for the type variable must be a subclass of the boundary type,
     see :pep:`484`.
 
-.. class:: ParamSpec(name, *, bound=None, covariant=False, contravariant=False)
+.. class:: ParamSpec(name, bound=None, covariant=False, contravariant=False)
```

Comment:
Hm. I would rather add the star to the code.

Reply:
Hmm after giving it some thought, I agree. I realized ParamSpec and TypeVar have different init signatures, (TypeVar has ``*bounds`` and ParamSpec doesn't). And PEP 612 states that 
> However, enforcing these constraints would require enough additional implementation work that we judged this extension to be out of scope of this PEP. Fortunately the design of ParamSpecs are such that we can return to this idea later if there is sufficient demand.

so to prevent any backwards incompatibility in the future, we should enforce it, so that if we _do_ add bounds in the future, existing code won't break.